.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "jsonapi-kotlin"]
+	path = jsonapi-kotlin
+	url = git@github.com:nimblehq/jsonapi-kotlin.git
```

Comment:
Please follow [this doc](https://github.com/nimblehq/jsonapi-kotlin#installation) instead. My way is outdated 🤡

Reply:
@suho updated it 🙏